app/assets/stylesheets/users.css.scss

```diff
@@ -24,3 +24,8 @@
     }
   }
 }
+
+.progress_chart .panel-body {
+  padding-top:0px;
```

Comment:
Properties should be ordered margin-top, padding-top<br>Colon after property should be followed by one space<br>`0px` should be written without units as `0`